src/server/stores/user-store.js

```diff
@@ -31,10 +34,27 @@ function validate(password, user) {
  * @returns {Promise.<object|null>} Promise resolving to the user, or null if one cannot be found
  */
 export function get(email) {
-  if (!fakeStore.hasOwnProperty(email)) {
-    return Promise.resolve(null);
-  }
-  return Promise.resolve(Object.assign({}, fakeStore[email]));
+    
+    if(config.db.isMocked){
+        if (!fakeStore.hasOwnProperty(email)) {
+            return Promise.resolve(null);
+        }
+        return Promise.resolve(Object.assign({}, fakeStore[email]));
+    }
+    
+    var query = "SELECT CONCAT(firstName,' ',lastName) AS name, photo AS avatar, email, password " +
```

Comment:
Could be `let`.

`var` is basically deprecated in ES6, since its strange functional-scoping makes no sense to anyone except javascript elders. Use `const` if the variable won't change, and `let` if it will.

Reply:
The more you know... I'll change it. 

BTW,  I fucking despise how JS will just spawn a null-valued variable out of thin air when you make a typo,  instead of having the compiler yell at you about an undefined identifier, like any decent self-respecting language would. Is there a way around that?

Reply:
(Strict mode)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Strict_mode]

Reply:
Ah, no, wait. I might have used a function name when I meant to use a string variable, and the function just got magically and silently casted to an empty string, instead of barfing at me.

Reply:
Is there a way to enable strict mode project-wide? (unless you think that's a bad idea)

Reply:
I can add a babel plugin that will automatically inject 'use strict' into all the compiled files. That's as close as we can get to project-wide strict mode. I'll add it to my list.

Reply:
Re. casting functions to strings: They should end up as string representations of the function, not the empty string. Try this in a browser console:

```
let foo = function() {};
console.log(`This is foo: ${foo}`);
```

One of my favourite things about javascript is your web browser has a great suite dev tools, including a full REPL.